### ISSUE TITLE
Logging exception information can be confusing

### ISSUE THREAD

**Issue by kimbell** [aspnet/Logging#446]
Looking at the sample application, how to log exception information can be rather confusing. 

The sample uses these lines:
```
_logger.LogCritical("Unexpected critical error starting application", ex);
_logger.LogError("Unexpected error", ex);
_logger.LogWarning("Unexpected warning", ex);
```
The last argument being the exception, one would think the exception information and stack trace was written to the logger. That doesn't happen. 

I had to change it so I was using a very different extension method. 
` _logger.LogCritical(23, ex, "some message");`
![logging](https://cloud.githubusercontent.com/assets/11733077/15804881/8089bfec-2b18-11e6-8699-db8b7bc89906.png)

Only when using a method that takes an event id as the first argument, the stack trace is displayed.

**Comment by BrennanConroy** [comment]
Looks like the sample hasn't been updated since some changes we made (including the API changes I made). I'll work on fixing it.

**Comment by BrennanConroy** [comment]
Fixed sample in ddc1409be0e34c6973570b2b7b3d1686925d3658
